best-practices-zarr.md

```diff
@@ -0,0 +1,549 @@
+      {"name": "b02", "eo:common_name": "blue"},
+      {"name": "b03", "eo:common_name": "green"},
+      {"name": "b04", "eo:common_name": "red"},
+    ]
+  }
+}
+```
+
+##### Multiple bands per data variable (band as dimension)
+
+In some cases, a single data variable may contain multiple bands along a specific dimension (e.g., spectral bands stored as the dimension of the array). In this case, the band objects SHALL reference indexes within a variable, with the `name` field indicating both the variable name and the dimension index or label.
+
+```json
+"assets": {
+  "reflectance": {
+    "href": "s3://bucket/path/data.zarr",
+    "type": "application/vnd+zarr; version=3",
+    "bands": [
+    {"name": "reflectance[band=blue]", "eo:common_name": "blue"},
+    {"name": "reflectance[band=green]", "eo:common_name": "green"},
+    {"name": "reflectance[band=red]", "eo:common_name": "red"},
+    ],
+    "cube:variables": {
+      "reflectance": {
+        "dimensions": ["y", "x", "band"],
+        "type": "data",
+      }
+    },
+    "cube:dimensions": {
+      "band": {
+        "type": "spectral",
+        "description": "Spectral bands",
+        "values": ["blue", "green", "red"]
+      }
+    }
+  }
+}
+```
+
+##### Bands in multiscales groups
+
+For multi-resolution data organized using the [multiscales convention](https://github.com/zarr-conventions/multiscales), bands are organized within resolution groups hosting data variables.
+
+The hierarchical structure of a multiscales Zarr store looks like this:
+
+```console
+dataset.zarr/
+      │
+      ├─► measurements/
+      │   │
+      │   └─► reflectance/         (multiscales group)
+      │        │
+      │        ├─► r10m/           (resolution level)
+      │        │    ├─► b02/       (array)
+      │        │    ├─► b03/       (array)
+      │        │    └─► b04/       (array)
+      │        │
+      │        ├─► r20m/           (resolution level)
+      │        │    └─► ...
+      │        │
+      │        ├─► r60m/           (resolution level)
+      │        │    └─► ...
+      │
+      └─► quality/
+```
+
+The `bands` array SHALL reference individual bands, and the `name` field SHOULD follow the same pattern as in the "Unique band per data variable" and "Multiple bands per data variable" cases, depending on how bands are organized within each resolution group.
+The key point is that the resolution group do not appear directly in the metadata but is implicitly hidden behind the asset href. It is up to the client to construct and select the correct path to access individual arrays based on the [layout](https://github.com/zarr-conventions/multiscales?tab=readme-ov-file#layout) given in the multiscales convention.
+
+```json
+"assets": {
+    "reflectance": {
+      "href": "s3://esa-zarr-sentinel-explorer-fra/tests-output/sentinel-2-l2a/S2A_MSIL2A_20251008T100041_N0511_R122_T32TQM_20251008T122613.zarr/measurements/reflectance",
```

Comment:
Similar to the suggestion above, I think the zarr store location should be consistent across these examples:
```suggestion
      "href": "s3://bucket/path/data.zarr/measurements/reflectance",
```